ansible/roles/schulcloud-server-h5p-proxy/templates/api-h5p-proxy-ingress.yml.j2

```diff
@@ -29,13 +29,13 @@ spec:
 {% endif %}
 {% endif %}
   rules:
-  - host: {{ DOMAIN }}
+  - host: {{ H5P_PROXY_PREFIX |default("h5p-") }}{{ DOMAIN }}
     http:
       paths:
-      - path: /api/v3/h5p-editor/
+      - path: /
         backend:
           service:
-            name: api-h5p-svc
+            name: api-h5p-proxy-svc
             port:
-              number: 4448
+              number: 80

```

Comment:
If you want to send the /api/v3/h5p-editor/ request over the proxy make the service name and port number configurable if it is nessesery

Reply:
Agreed